graph/src/runtime/mod.rs

```diff
@@ -36,6 +36,18 @@ pub trait AscType: Sized {
     }
 }
 
+impl<T> AscType for std::marker::PhantomData<T> {
```

Comment:
I don't know if this is the best place to put this implementation though.

Reply:
Here is fine, but please add a comment that this is only to `derive(AscType)` for structs that contain `PhantomData`, and it is not really an asc type.

Reply:
Done ✅